frb_dart/pubspec.yaml

```diff
@@ -3,9 +3,20 @@ description: High-level memory-safe binding generator for Flutter/Dart <-> Rust
 version: 1.41.1
 repository: https://github.com/fzyzcjy/flutter_rust_bridge
 environment:
-  sdk: '>=2.14.0 <3.0.0'
+  sdk: ">=2.14.0 <3.0.0"
 dependencies:
+  args: ^2.3.1
+  fixnum: ^1.0.1
+  js: ^0.6.4
   meta: ^1.3.0
+  path: any
```

Comment:
uber-nit: Oops "any" does not sound like a great constraint. What about `^1.8.2` or sth like that

Reply:
I remember this conflicts with some other library, I'll try to resolve it later.

Reply:
Anyway "any" is too loose. Maybe `^x.x.x` or `>= x.x.x`